examples/go/sb-test/README.md

```diff
@@ -37,7 +37,8 @@ This is not an official Google product.
 
 * `/test-storage` - lists all buckets in the given project
 * `/test-pubsub` - returns the name of the first topic in the project
-* `/test-cloudsql` - tests the connection to the first bound database
+* `/test-cloudsql-mysql` - tests the connection to the first bound mysql database
+* `/test-cloudsql-postgres` - tests the connection to the first bound postgres database
```

Comment:
This url was just outdated so I updated it. It isn't super related to this change.